CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [3.1.0](https://github.com/functionalone/serverless-iam-roles-per-function/compare/v3.0.2...v3.1.0) (2020-12-17)
+
+
+### Features
+
+* permission boundary ([fb21075](https://github.com/functionalone/serverless-iam-roles-per-function/commit/fb210756744fca4a0b736c35c97b874c4374741a))
```

Comment:
Please update url with `https://github.com/functionalone/serverless-iam-roles-per-function/pull/68`.

Reply:
I can squash my commits in a way to meet the guide, but I think this PR will be closed when I delete the commits to create one with all. Do you have any suggestions on how to do this?

Reply:
You can squash and force push your commits. Pull request should be open and updated.

Reply:
done